sonarr/notification.go

```diff
@@ -0,0 +1,159 @@
+	OnHealthIssue                         bool                 `json:"onHealthIssue"`
+	OnApplicationUpdate                   bool                 `json:"onApplicationUpdate"`
+	SupportsOnGrab                        bool                 `json:"supportsOnGrab"`
+	SupportsOnDownload                    bool                 `json:"supportsOnDownload"`
+	SupportsOnUpgrade                     bool                 `json:"supportsOnUpgrade"`
+	SupportsOnRename                      bool                 `json:"supportsOnRename"`
+	SupportsOnSeriesDelete                bool                 `json:"supportsOnSeriesDelete"`
+	SupportsOnEpisodeFileDelete           bool                 `json:"supportsOnEpisodeFileDelete"`
+	SupportsOnEpisodeFileDeleteForUpgrade bool                 `json:"supportsOnEpisodeFileDeleteForUpgrade"`
+	SupportsOnHealthIssue                 bool                 `json:"supportsOnHealthIssue"`
+	SupportsOnApplicationUpdate           bool                 `json:"supportsOnApplicationUpdate"`
+	IncludeHealthWarnings                 bool                 `json:"includeHealthWarnings"`
+	ID                                    int64                `json:"id"`
+	Name                                  string               `json:"name"`
+	ImplementationName                    string               `json:"implementationName"`
+	Implementation                        string               `json:"implementation"`
+	ConfigContract                        string               `json:"configContract"`
+	InfoLink                              string               `json:"infoLink"`
+	Tags                                  []int                `json:"tags"`
+	Fields                                []*starr.FieldOutput `json:"fields"`
+}
+
+// GetNotifications returns all configured notifications.
+func (s *Sonarr) GetNotifications() ([]*NotificationOutput, error) {
+	return s.GetNotificationsContext(context.Background())
+}
+
+// GetNotificationsContext returns all configured notifications.
+func (s *Sonarr) GetNotificationsContext(ctx context.Context) ([]*NotificationOutput, error) {
+	var output []*NotificationOutput
+
+	req := starr.Request{URI: bpNotification}
+	if err := s.GetInto(ctx, req, &output); err != nil {
+		return nil, fmt.Errorf("api.Get%s): %w", &req, err)
+	}
+
+	return output, nil
+}
+
+// GetNotification returns a single notification.
+func (s *Sonarr) GetNotification(notificationID int) (*NotificationOutput, error) {
+	return s.GetNotificationContext(context.Background(), notificationID)
+}
+
+// GetNotificationContext returns a single notification.
+func (s *Sonarr) GetNotificationContext(ctx context.Context, notificationID int) (*NotificationOutput, error) {
+	var output NotificationOutput
+
+	req := starr.Request{URI: path.Join(bpNotification, fmt.Sprint(notificationID))}
+	if err := s.GetInto(ctx, req, &output); err != nil {
+		return nil, fmt.Errorf("api.Get%s): %w", &req, err)
```

Comment:
another... looks like you missed a bunch.

Reply:
yep, it seems I did few find and replace with wrong replacement